src/components/v5/common/ActionSidebar/partials/consts.tsx

```diff
@@ -5,11 +5,11 @@ import { ACTION } from '~constants/actions';
 
 import { CardSelectOption } from '../../Fields/CardSelect/types';
 
-export const DISTRIBUTION_METHOD = {
-  Equal: 'equal',
-  Unequal: 'unequal',
-  ReputationPercentage: 'reputation-percentage',
-} as const;
+export enum DISTRIBUTION_METHOD {
```

Comment:
Please could you ensure the casing in the names of enums and their members is exactly as in the issue description? This enum should become `DistributionMethod`